File: Iniciante/1080 - Maior e Posição/Python/1080.py
def main():
    numero = 0
    maior_valor = 0
    posicao = 0
    contador = 1

    while contador <= 100:
        numero = int(input())

        if numero > maior_valor:
            maior_valor = numero
            posicao = contador

        contador += 1

    print(maior_valor)
    print(posicao)


main()

File: Iniciante/1080 - Maior e Posição/JavaScript/1080.js
function main() {
  var input = require("fs").readFileSync("/dev/stdin", "utf8");
  var lines = input.split("\n");

  console.log(maiorValor(lines));
}

function maiorValor(l) {
  let lista = l.map((item) => parseInt(item, 10));
  let numero = 0;
  let valorMaior = 0;
  let posicao = 0;

  for (let i = 0; i <= 100; i++) {
    numero = lista[i];

    if (numero >= valorMaior) {
      valorMaior = numero;
      posicao = i + 1;
    }
  }

  return `${valorMaior}\n${posicao}`;
}

main();
